src/ICal/ICal.php

```diff
@@ -2112,12 +2121,12 @@ public function hasEvents()
      * problem for events on, during, or after 29 Jan 2038.
      * See https://en.wikipedia.org/wiki/Unix_time#Representing_the_number
      *
-     * @param  string $rangeStart
-     * @param  string $rangeEnd
+     * @param  string|null $rangeStartS
+     * @param  string|null $rangeEndS
      * @return array
      * @throws \Exception
      */
-    public function eventsFromRange($rangeStart = false, $rangeEnd = false)
+    public function eventsFromRange($rangeStartS = null, $rangeEndS = null)
```

Comment:
Why the variable name changes?

Reply:
Overwriting function parameters (that are not passed by reference for output) is bad practice. It's even worse practice when they are assigned an entirely different type of value, which was the case here - an input string value was overwritten with a DateTime object.. This is code cleanup so modern linters won't complain about the variable types being managed poorly.